README.md

```diff
@@ -2,63 +2,42 @@
 
 # Endgame Malware BEnchmark for Research
 
-The ember dataset is a collection of 1.1 million sha256 hashes from PE files that were scanned sometime in 2017. This repository makes it easy to reproducibly train the benchmark model, extend the provided feature set, or classify new PE files with the benchmark model.
+The EMBER dataset is a collection of features from PE files that serve as a benchmark dataset for researchers. The EMBER2017 dataset contained features from 1.1 million PE files scanned in 2017 and the EMBER2018 dataset contains features from 1 million PE files scanned in 2018. This repository makes it easy to reproducibly train the benchmark models, extend the provided feature set, or classify new PE files with the benchmark models.
```

Comment:
"1.1 million PE files scanned in 2017" ==> . "1.1 million PE files scanned in or before 2017"
"1 million PE files scanned in 2018" ==> . "1 million PE files scanned in or before 2018"